### ISSUE TITLE
v2 update?

### ISSUE THREAD

**Issue by Blugil** [canbeardig/Spicetify-Orchis-Colours#1]
Hey it doesn't appear that this theme works with the v2 spicetify update, any chance you would update that soon? Thank you!

**Comment by canbeardig** [comment]
Yeah I've been delaying the v2 update as Spotify on Linux hasnt' been upgraded to version 1.1.59 but I'm planning to update it this week once I boot the Windows.

**Comment by Blugil** [comment]
Sweet thanks so much, love this time.
